ci.sh

```diff
@@ -1,18 +1,11 @@
 #!/bin/bash
 set -ex
 
-# Fixed `minirust-tooling` commit, we need to bump this occasionally.
-TOOLING_COMMIT="b1b41a1"
+# Fixed specr-transpile version
+VERSION="0.1.3"
 
-# where to check out the tooling
-TOOLING_DIR="$HOME/minirust-tooling"
+rustup default nightly-2023-02-12
```

Comment:
`specr-transpile` needs this. Is there a way to fix the rust nightly version inside of the specr-transpile crate?
rust-toolchain.toml is ignored by crates.io

Reply:
Nice. :)